EDU-17770 - Password Protection in WebOps (#2571)
vtexdocs/dev-portal-content: docs/faststore/docs/getting-started/webops/dashboard.mdx

> ⚠️ If your domain isn't listed, there might be an issue with the configuration or DNS settings that you need to address before proceeding with the go-live. If you follow the [external DNS settings](https://developers.vtex.com/docs/guides/faststore/go-live-1-configuring-external-dns) guide correctly, and the problem persists, open a ticket with [VTEX Support](https://help.vtex.com/en/support).

### Password protection

> The **Password protection** section allows you to restrict access to your preview environments and production domain by requiring username and password authentication. This is useful for protecting internal testing environments, staging builds, or production domains from unauthorized access during development or initial launch phases.

Author: ```suggestion
The **Password protection** section allows you to restrict access to your preview environments and production domain by requiring password authentication. This is useful for protecting internal testing environments, staging builds, or production domains from unauthorized access during development or initial launch phases.
```

> ![password-protection](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@EDU-17770/images/password-protection.png)

Author: ```suggestion
![password-protection](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@EDU-17770/images/password-protection-01.png)
```

### Password protection

> ℹ️ Password protection is in closed beta, which means that only selected customers can access it. If you're interested in implementing it in the future, please contact our [Support](https://support.vtex.com/hc/en-us) team.

The **Password protection** section allows you to restrict access to your preview environments and production domain by requiring password authentication. This is useful for protecting internal testing environments, staging builds, or production domains from unauthorized access during development or initial launch phases.

![password-protection](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@EDU-17770/images/password-protection-01.png)

In the **Require password for** section, there are the following:

* **Default domains (*.vtex.app):** Applies password protection to testing URLs that use VTEX's default `vtex.app` domain. Select this option to restrict access to internal preview environments, branch previews, and other non-public links generated during development.

> * **Custom domains (e.g. www.mystore.com):** Applies password protection to your public-facing store domain configured in FastStore (for example, www.mystore.com). Select this option when you want to temporarily gate access to your production site or a custom staging domain during soft launches, QA, or controlled rollouts. To enable password protection for custom domains, open a ticket with [VTEX Support](https://help.vtex.com/en/support).

Author: ```suggestion
* **Custom domains (e.g. www.mystore.com):** Applies password protection to your public-facing store domain configured in FastStore (for example, www.mystore.com). Select this option when you want to temporarily gate access to your production site or a custom staging domain during soft launches, QA, or controlled rollouts.

  > ℹ️ Password protection for custom domains isn't enabled by default. If you want to restrict access for your store domains configured in FastStore, you must open a ticket with [VTEX Support](https://help.vtex.com/en/support).
```

The **Password protection** section allows you to restrict access to your preview environments and production domain by requiring password authentication. This is useful for protecting internal testing environments, staging builds, or production domains from unauthorized access during development or initial launch phases.

![password-protection](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@EDU-17770/images/password-protection-01.png)

In the **Require password for** section, there are the following:

* **Default domains (*.vtex.app):** Applies password protection to testing URLs that use VTEX's default `vtex.app` domain. Select this option to restrict access to internal preview environments, branch previews, and other non-public links generated during development.
* **Custom domains (e.g. www.mystore.com):** Applies password protection to your public-facing store domain configured in FastStore (for example, www.mystore.com). Select this option when you want to temporarily gate access to your production site or a custom staging domain during soft launches, QA, or controlled rollouts.

>   > ℹ️ Password protection for custom domains isn't enabled by default. If you want to restrict access for your store domains configured in FastStore, you must open a ticket with [VTEX Support](https://help.vtex.com/en/support).

Author: ```suggestion
  > ℹ️ Password protection for **custom domains** isn't enabled by default. If you want to restrict access for your store domains configured in FastStore, you must open a ticket with [VTEX Support](https://help.vtex.com/en/support).
```

![password-protection](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@EDU-17770/images/password-protection-01.png)

In the **Require password for** section, you can configure the following:

| Domain type | Description | Availability |
| ----------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
| **Default domains (*.vtex.app)**          | Applies password protection to testing URLs that use VTEX's default `vtex.app` domain. Select this option to restrict access to internal preview environments, branch previews, and other non-public links generated during development. | ✅ Available by default. |

> | **Custom domains (e.g. www.mystore.com)** | Applies password protection to your public-facing store domain configured in FastStore (for example, www.mystore.com). Select this option when you want to temporarily gate access to your production site or a custom staging domain during soft launches, QA, or controlled rollouts. | ⚠️ Not enabled by default. You must open a ticket with [VTEX Support](https://help.vtex.com/en/support). |

Author: ```suggestion
| **Custom domains (e.g. `www.mystore.com`)** | Applies password protection to your public-facing store domain configured in FastStore (for example, `www.mystore.com`). Select this option when you want to temporarily gate access to your production site or a custom staging domain during soft launches, QA, or controlled rollouts. | ⚠️ Not enabled by default. You must open a ticket with [VTEX Support](https://help.vtex.com/en/support). |
```
